config/config.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Is this config should be empty?

Reply:
Yes, the config with secrets usually contains secret values, tables/databases/etc names. 
If user need to make any changes to the project, such as the address, table name, or database, they can do so by making changes to the config.